lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2114,6 +2114,9 @@ Instruction *SPIRVToLLVM::transEnqueueKernelBI(SPIRVInstruction *BI,
     SPIRVConstant *NE = static_cast<SPIRVConstant *>(NumEvents);
```

Comment:
I think we could move below condition here, like:
`if (EventRet->getOpCode() == OpConstantNull && NumEvents->getOpCode() == OpConstant)`
As it doesn't make sense to set HasEvents to false and then back to true in the next condition.

Reply:
This comment refers to line 2113.

Reply:
done

Reply:
LGTM, thanks!